main.js

```diff
@@ -40,7 +40,9 @@ function createOverlayWindow() {
     settings.set('position', {x, y});
   });
 
-  if (debug) overlayWindow.webContents.openDevTools(DEVTOOL_OPTIONS);
+  overlayWindow.webContents.openDevTools();
+
+  if (debug) overlayWindow.webContents.openDevTools();
```

Comment:
This might be an error

Reply:
My bad, I found out (at least on my PC) that using the `DEVTOOL_OPTIONS` parameter doesn't show the dev tool. I'll remove line 43 tho, that was my bad.